webSites/webSites-war/src/main/java/org/silverpeas/components/websites/servlets/design/SiteDesignActionHandler.java

```diff
@@ -0,0 +1,660 @@
+    }
+
+    return "/webSites/jsp/design.jsp?Action=design&path=" + currentPath + "&Id=" + id;
+  }
+
+  private void classifySite(final WebSiteSessionController controller, final HttpRequest request,
+      final String id) throws WebSitesException {
+    // CLASSIFY SITE -------------------------------------------------------------
+    // = en cas de new Site ou de classifySite
+    String listeTopics = request.getParameter(WebSitesRequestRouter.TOPIC_LIST_PARAM);
+
+    request.setAttribute("Site", controller.getSessionSite());
+
+          /* publications : classer le site dans les themes cochés */
+    ArrayList<String> arrayToClassify = new ArrayList<>();
+    boolean publish = false;
+
+    ArrayList<String> arrayTopic = new ArrayList<>();
+    int begin = 0;
+    int end = 0;
+    end = listeTopics.indexOf(',', begin);
+    String idTopic = null;
+    while (end != -1) {
+      idTopic = listeTopics.substring(begin, end);
+
+      begin = end + 1;
+      end = listeTopics.indexOf(',', begin);
+
+      arrayTopic.add(idTopic);
+      publish = true;
+    }
+    controller.updateClassification(id, arrayTopic);
+
+    arrayToClassify.add(id);
+    if (publish) {
+      controller.publish(arrayToClassify); // set etat du site a 1
+    } else {
+      controller.dePublish(arrayToClassify); // set etat du site a 0
+    }
+  }
+
+  private void deletePageInSite(final WebSiteSessionController controller, final HttpRequest request,
+      final String currentPath) throws WebSitesException {
+    // DELETE PAGE -------------------------------------------------------------
+    // = null la premiere fois, puis = nom du repertoire courant
+    String name = request.getParameter("name");
+
+          /* Supprimer la page */
+    controller.deleteFile(currentPath + "/" + name);
+
+    request.setAttribute("Site", controller.getSessionSite());
+  }
+
+  private void renamePageInSite(final WebSiteSessionController controller, final HttpRequest request,
+      final String currentPath) throws WebSitesException {
+    // RENAME PAGE -------------------------------------------------------------
+    // = null la premiere fois, puis = nom du repertoire courant
+    String name = request.getParameter("name");
+    // = changement de noms des fichiers et repertoires
+    String newName = request.getParameter("newName");
+
+          /* Modifier le nom du fichier */
+    controller.renameFile(currentPath, name, newName);
+
+    request.setAttribute("Site", controller.getSessionSite());
+  }
+
+  private void addPageInSite(final WebSiteSessionController controller, final HttpRequest request,
+      final String currentPath) throws WebSitesException {
+    // = rempli la premiere fois a la creation, puis toujours null
+    String nomPage = request.getParameter(WebSitesRequestRouter.PAGE_NAME_PARAM);
+
+    // ADD PAGE -------------------------------------------------------------
+    String code = request.getParameter("Code"); // = code de la page a parser
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ATrailingCommentCheck)